### DISCUSSION TITLE
How to read a saved pdb instead of retrieving from rcsb

### DISCUSSION THREAD

**Discussion by ssunidhi** [biotite-dev/biotite#286] [Q&A] [answered]
Hey I just went through your superimposition code. A quick query regarding that.. how do I read a local pdb file and still get all the functions to perform (instead of retrieving from rcsb and then reading them)?

**Comment by padix-key** [comment, marked as answer]
Instead of `pdbx.get_structure(pdbx.PDBxFile.read(rcsb.fetch("1JEY", "cif")))` you can use the `structure.io.pdb.PDBFile` class, e.g.:
```python
pdb_file = pdb.PDBFile.read("path/to/your/file")
structure = pdb.get_structure(pdb_file, model=1)
```
